packages/generator-single-spa/src/validate-naming.js

```diff
@@ -0,0 +1,15 @@
+const BEGIN_WITH_LETTER = /^[a-z]/;
+const ALLOWED_CHARACTERS = /^[a-z0-9\-]+$/;
```

Comment:
should we allow capital characters, too?

Reply:
I debated that as well. I don't see anything wrong with it since 1) for single-spa its mostly a convention rather than a requirement, and 2) that is permitted in package.json naming. The only counterargument is that its gone this long without being something that users have requested/reported. But it should be pretty easy to add if we do want it.

Reply:
Yeah I say let's add it - no reason to enforce lowercase i think